src/common/process/ProcessUtils.cpp

```diff
@@ -142,9 +142,12 @@ pid_t ProcessUtils::maxPid() {
 
 
 StatusOr<std::string> ProcessUtils::runCommand(const char* command) {
+    std::string cmd;
+    cmd += command;
+    cmd += " 2>&1";
```

Comment:
why put it in command by default.  I don't think it is really a good idea.

Reply:
With checking existing code, it can be removed.